src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellToolbarTracker.cs

```diff
@@ -392,7 +394,7 @@ protected virtual async void UpdateLeftBarButtonItem(Context context, AToolbar t
 			DrawerArrowDrawable icon = null;
 			bool defaultDrawerArrowDrawable = false;
 
-			var tintColor = Colors.White;
+			var tintColor = Shell.GetForegroundColor(page) ?? Colors.White;
```

Comment:
Does this make it match the behavior on iOS and Android?

Reply:
Yes, here's an iOS sample:

![Simulator Screenshot - iPhone 15 Pro Max - 2024-06-10 at 00 41 23](https://github.com/dotnet/maui/assets/42434498/76638afe-6ab0-4a91-a428-4c3b0d1f32e6)

Reply:
Is the icon misaligned on iOS?

Reply:
I used the default icon that has been included in the test app. It is too big, so it is why it looks like this. It's a different bug but not related to this change